### ISSUE TITLE
Can't hide individual points on a Series

### ISSUE THREAD

**Issue by sushi19921992** [oxyplot/oxyplot#1571]

**Comment by sushi19921992** [comment]
@Jonarw added details

**Comment by Jonarw** [comment]
Which series?
What's stopping you from just removing these points from the series?
Did you have a look at `Axis.FilterMinValue` and `Axis.FilterMaxValue`?

**Comment by sushi992** [comment]
Working on a Scatter Series. Essentially we have a region that we can adjust to show certain points on a series. We need to hold that information in our series - I suppose we can store the actual points in a dataset and remove the points from the region selected, but we don't want to have to keep doing that all the time. The region is user defined and can be changed from 0-X, or 4-6, back to 0-X as an example. I've just seen `FilterMinValue` and max. There's not much documentation on them, do they just simply hide the points within that range, then once set back again they'll appear again?

**Comment by Jonarw** [comment]
Yes, if you set `FilterMinValue` to a certain value, only points with a larger value on that axis should be displayed. Once you set `FilterMinValue` to `double.MinValue` again, all values should be displayed again (of course vice versa for `FilterMaxValue`).

More complicated filters can be set up via `Axis.FilterFunction`.

For reference:
https://github.com/oxyplot/oxyplot/blob/7329013d6f48331e42b81c4315cd13fb5a82bb8f/Source/OxyPlot/Axes/Axis.cs#L771

**Comment by sushi19921992** [comment]
Thanks @Jonarw , I have tried that, and have set filter min to 2 and filter max to 4 for example, but for some reason, the absolute x min and max values change too? i.e. initially it would go from 0 to 6, then once applying filter min and max, the absolute max goes to 100, cramping everything up. Is this meant to happen?

**Comment by sushi19921992** [comment]
@VisualMelon i think i read you've seen something like this happen too?

**Comment by sushi19921992** [comment]
Hmm it seems if you call InvalidatePlot(false) it works properly. Not sure I fully understand why but hey hum. One quick thing though. I have a fit line that goes from a very large negative X to a very large positive X. Obviously, when doing the above filter, you can't see the line anymore. I know there's some documentation on more complex filtering, but could you guys point me in the direction where I can read up about excluding a series? My series for the fit line has the title "Fit", so if I can find a series with the title "Fit" then i can try and somehow exclude it from the filtering.

**Comment by sushi19921992** [comment]
Basically, I only want that filter to affect all series apart from one, the fit line

**Comment by VisualMelon** [comment]
I've been using this library for years, and I use a custom build at work, so I've seen a lot I can't really remember which may or may not apply ;)

Could you make the fit line an annotation? I don't think they get filtered, and it sounds like a better fit for what you describe.

**Comment by sushi19921992** [comment]
You know @VisualMelon that's not such a bad idea! IF annotations don't get filtered out..... 👀  I'll try it and get back to you :)

**Comment by sushi19921992** [comment]
Ok, so that works, however I notice, when I switch tabs (say I have a plotview on one tab and another plotview on another tab in a tab control) it defaults back to 0-100 on the x axis :(...
